tests/airportDetailsProvider.test.js

```diff
@@ -0,0 +1,23 @@
+const {airportInformation} = require('../src/airportDetailsProvider.js');
+describe('airportInformation that returns details of airport', () => {
+    it('should return details of airport when code given', ()=>{
+        const testInformation = airportInformation('DEL');
+        expect(testInformation).toStrictEqual({ name: 'Indira Gandhi International Airport', city: 'New Delhi', country: 'India'});
```

Comment:
Better to mock `const airportDetails = require('.../data/airports.json');` as test shouldn't be dependent on values in the .json file, what if DEL is removed permanently from file. The tests would fail, purpose here is to test the logic of `airportInformation`